src/main/webapp/user-page.html

```diff
@@ -60,32 +50,58 @@ <h1 class="display-4" id="page-title">My Page</h1>
           <input type="submit" value="Submit" />
         </form>
       </div>
-    </div>
+    </div> -->
```

Comment:
Need this code for future reference?

Reply:
Yes. This is to fill in the about-me or any other relevant profile information.

Reply:
Alright! Thanks for clearing it up 😄